ci-operator/step-registry/openshift/microshift/e2e/footprint-and-performance/openshift-microshift-e2e-footprint-and-performance-workflow.yaml

```diff
@@ -3,6 +3,7 @@ workflow:
   steps:
     allow_best_effort_post_steps: true
     pre:
+      - ref: openshift-microshift-includes
```

Comment:
We also need this in https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/microshift/infra/aws/pre/openshift-microshift-infra-aws-pre-chain.yaml as the first entry.

Reply:
Can you explain why? I was planning to only add this step to workflows, not other chains....